src/documentation/pages/components/marketing-icon/examples/_icons.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+  <ComponentExample title="Icons" id="icons" padding="-p--0">
```

Comment:
This should not be used here as there is no need for it. The example is very different from the majority of component examples.
The example does not have any tabs and adding `v-if=""` to tabs complicates the reusable component `ComponentExample` for no added value.
Please, remove the `ComponentExample` and use a simple `div` or `Wrapper` instead.

Reply:
`ComponentExample` was created as a reusable component to make it easier to manage tabs, slots, etc. But its use is optional.

Reply:
Ah gotcha! Didn't consider it as optional for some reason